onnx/test/runtime_backend_test.py

```diff
@@ -0,0 +1,609 @@
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
+# pylint: disable=C0415,R0912,R0913,R0914,R0915,W0640,W0703
+
```

Comment:
A few lines documenting the scope of this file would help. My understanding is that this is for the backend\test\case\node tests? Does it also include backend\test\case\model tests?

Reply:
Yes, it runs all the backend test with the python runtime and checks that it produces the expected outputs.

Reply:
Done